src/components/InlineInput.js

```diff
@@ -8,6 +8,7 @@ type Props = {|
   onChange: (SyntheticInputEvent<HTMLInputElement>) => void,
   type: string,
   value: string | string[] | number,
+  min?: number,
```

Comment:
Flow can infer if a prop is optional if it's declared in `defaultProps`. From [the docs](https://flow.org/en/docs/react/components/#toc-using-default-props):

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

However, thread carefully – dealing with default props quickly signs you up for the battle between ESlint and Flow. Because I fought in that war several times before I will spare you from [all.](https://github.com/facebook/flow/issues/6644) [the.](https://github.com/yannickcr/eslint-plugin-react/issues/1212) [boring.](https://github.com/facebook/flow/issues/4871) parts:

1. Add the default prop: `min: undefined`
2. Flow now complains that `undefined is not a number`. This gives you two options, either make the prop nullable again by adding `min?: number` (which is kinda what they said we didn't need 🤷‍♀️) or make `undefined` a valid value by adding `min: number | void`. 

After a lot of scars I usually end up with the latter:

```suggestion
  min: number | void,
```

Reply:
Thanks for the detailed explanation! Today I learned!